sample.env

```diff
@@ -0,0 +1,3 @@
+PORT=
+DB_URI=
```

Comment:
https://github.com/freeCodeCamp/boilerplate-project-library/blob/7692c6ab272e1f87cbaf3c7df89e5dbd89de7afb/routes/api.js#L14

We've named this one differently in @SaintPeter's pull request - should we update that PR to reflect this variable name?

Reply:
Hi @nhcarrigan, thanks for catching that. I just noticed this myself.

Either is totally fine, and learners can change it themselves later if they want.

I'll go ahead and change it here to reflect what's in PR #5.